web-app/js/portal/cart/DownloadPanelItemTemplate.js

```diff
@@ -86,10 +86,33 @@ Portal.cart.DownloadPanelItemTemplate = Ext.extend(Ext.XTemplate, {
     },
 
     _downloadButton: function(collection) {
-        this._createDownloadButton.defer(1, this, [collection]);
+
+        if (collection.downloadStatus == 'requested') {
+            this._createDownloadingLabel.defer(1, this, [collection]);
+        }
+        else {
+            this._createDownloadButton.defer(1, this, [collection]);
+        }
+
         return '';
     },
 
+    // It's *actually* a button, but we're using it as a label here...
+    _createDownloadingLabel: function(collection) {
+        var elementId = this._getButtonId(collection, 'downloadButtonId');
+
+        Ext.fly(elementId).update("");
+
+        new Ext.Button({
+            text: OpenLayers.i18n('downloadStatusRequested'),
+            icon: 'images/spinner_orange_background.gif',
```

Comment:
@pmbohm the only font awesome spinner is this one: fa-spinner [&#xf110;] which looks a bit different.

I think I might leave this as is for now, as:

1. I don't want to change every spinner as part of this PR
2. the fa-spinner doesn't seem to actually spin - it's just static (at least where I'm using it).